lib/src/widgets/search_bar.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+
+import '../constants/strings.dart';
+import '../constants/text_style.dart';
+import '../models/coins.dart';
+import '../screens/detail_screen/detail_screen.dart';
+
+class SearchBar extends StatefulWidget {
+  List<Coins> datalist;
```

Comment:
because it is List<Coins> show may we should name it coinList?

Reply:
i fixed